### ISSUE TITLE
Use same VM template across all computes.

### ISSUE THREAD

**Issue by vint66** [GNS3/gns3-server#1657]
As for now in order to use remote server in addition to main server we have to create VM template per compute node. It means same VM should be duplicated per compute node.

Could you improve that part?

Basically it should be one template only but should be an option to run it on particular compute node, similar how you did for cloud object.

This will improve scale out options.

**Comment by grossmj** [comment]
I agree, we already have an issue opened for this: https://github.com/GNS3/gns3-server/issues/1500

The only hard part is to make sure images are on the configured computes.

**Comment by vint66** [comment]
Hi Jeremy,
Thank you for reply. Here is some simple idea to get it done.

Currently if you feed existing controller’s config it will have all templates even though no images (it is just verifying image existence in time of creation).
For satellite servers it could be the same - just synch full config (or only templates selected - this will give more granular control) and if image missing it will raise an error in time of node creation.

**Comment by grossmj** [comment]
Related issue: https://github.com/GNS3/gns3-server/issues/1553

**Comment by grossmj** [comment]
Template can be used across all computes on the back-end. Now we need to implement the possibility to change computes on GUI side.
